test/unit/auth/project-config.spec.ts

```diff
@@ -94,6 +96,14 @@ describe('ProjectConfig', () => {
         }).to.throw('"RecaptchaConfig.emailPasswordEnforcementState" must be either "OFF", "AUDIT" or "ENFORCE".');
       });
 
+      it('should throw on non-boolean useAccountDefender attribute', () => {
+        const configOptionsClientRequest = deepCopy(updateProjectConfigRequest) as any;
+        configOptionsClientRequest.recaptchaConfig.useAccountDefender = 'no';
```

Comment:
Nit: feel free to ignore. You can also test for few more different types in addition to a `string` here.

```
const invalidUseAccountDefender = [null, NaN, 0, 1, '', 'a', [], [1, 'a'], {}, { a: 1 }, _.noop];
    invalidUseAccountDefender.forEach((useAccountDefender) => {
      it(`should throw given invalid useAccountDefender parameter: ${JSON.stringify(useAccountDefender)}`, () => {
```